app/services/doc_auth/error_generator.rb

```diff
@@ -103,11 +103,86 @@ def get_image_metric_errors(processed_image_metrics)
     end
   end
 
+  class DocAuthErrorHandler < ErrorHandler
+    def handle(response_info, known_alert_error_count)
+      get_doc_auth_errors(response_info, known_alert_error_count)
+    end
+
+    private
+
+    def get_doc_auth_errors(response_info, known_error_count)
+      doc_auth_error_messages = get_doc_auth_error_messages(response_info)
+      liveness_enabled = response_info[:liveness_enabled]
+
+      # don't worry about unknown alert errors here
+      if known_error_count < 1
+        return nil
+      elsif known_error_count == 1
+        process_single_doc_auth_error(doc_auth_error_messages)
+      elsif known_error_count > 1
+        # Simplify multiple errors into a single error for the user
+        consolidate_multiple_doc_auth_errors(doc_auth_error_messages, liveness_enabled)
+      else
+        # default fall back
+        ErrorResult.new(error, side)
+      end
```

Comment:
```suggestion
      return if known_error_count < 1

      doc_auth_error_messages = get_doc_auth_error_messages(response_info)
      liveness_enabled = response_info[:liveness_enabled]

      # don't worry about unknown alert errors here
      if known_error_count == 1
        process_single_doc_auth_error(doc_auth_error_messages)
      else known_error_count > 1
        # Simplify multiple errors into a single error for the user
        consolidate_multiple_doc_auth_errors(doc_auth_error_messages, liveness_enabled)
      end
```

Reply:
I see. Your solution is more straightforward! Will commit and see if the tests pass.